packages/clients/meldemichel/src/mapConfigurations.ts

```diff
@@ -161,15 +164,17 @@ const mapConfigurations = {
             // TODO doesn't work atm; no coordinate source
             geometry: false,
             window: true,
-            properties: {
-              filename: 'Name of file',
-            },
+            properties: { filename: 'Name of file' },
           },
         },
         coordinateSources: [], // to be set in addPlugins.ts
       },
       pins: commonPins,
       reverseGeocoder,
+      meldemichelAfmButton: {
+        displayComponent: true,
+        afmUrl,
```

Comment:
Wouldn't just `url` suffice?

Reply:
It's settable from the outer `createMaps` and called `afmUrl` there. I figured the prefix would make more clear what it is. Just naming it `url` in `createMap` could be misleading, and I don't think we should have multiple names for one thing.

Reply:
In `core` (regarding layer configuration), in `addressSearch` (regarding search configuration) and in `reverseGeocoder` (regarding WPS configuration) it is always simply named `url` - why should this be different?

The name in `createMap` should stay this way, because the argument for clear usage can be made there.  
I'd say it is clear enough that `afmUrl` changes the value of `meldemichel.afmButton.url`; the lastest, when one looks in the README.md.

Reply:
Renaming `url: afmUrl` considered harmful.

Reply:
Talked this through ☎️